### PR TITLE
Add info messages to git-flow tests

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Adds info messages to git-flow tests to improve reading logs in OpenShift CI infrastructure
* Adds a style and highlighting info messages to improve readable logs in OpenShift CI infrastructure

### Screenshot/screencast of this PR
Screenshot-4
Screenshot-5
Screenshot-3
Screenshot-2
Screenshot-1

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/22902

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ set -ex
 # only exit with zero if all commands of the pipeline exit successfully
 set -o pipefail
 
+echo -e "\e[1;3;4;32m[INFO] OpenShift CI infrastructure is ready.\nTest is run.\e[0m"
```
Comment (vinokurig) [inline .ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh:19]: `Test is run` -> `Running test`?

Comment (artaleks9) [reply .ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh:19]: Yeah.. 
Maybe `Test is running` is more preferable, you mean this ?

Comment (vinokurig) [reply .ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh:19]: Not a big deal but, `Test is running` sounds like you request a current status and the response is: the test is running. However, this log line is more like a next step indicator, so I think `Running test(s)` sounds more suitable.

Comment (artaleks9) [reply .ci/openshift-ci/test-azure-no-pat-oauth-flow-raw-devfile-url.sh:19]: OK, got it

**Comment by vinokurig** [comment]
@artaleks9 I think you should align all messages to be equal.

**Comment by artaleks9** [comment]
So..Unfortunately the OpenShift CI can't allow to apply a style and highlighting as on screenshots, so info messages will be marked with special symbols, like:
```
"======= [INFO] OpenShift CI infrastructure is ready. Running test. ======="

"------- [INFO] Initialize user namespace -------"
"------- [INFO] Setup Personal Access Token Secret -------"


"####### [ERROR] Cause possible: lost connection to pod, this is an infrastructure problem. Try to rerun the test. #######"
```

**Review by vinokurig — APPROVED** [review]

**Pull request merged**